src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -621,14 +624,27 @@ private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSy
             else if (node.UsingKeyword != default)
             {
                 kind = LocalDeclarationKind.UsingVariable;
-            }
+                BoundLocalDeclaration[] boundDeclarations = new BoundLocalDeclaration[variableCount];
+                int i = 0;
+                foreach (var variableDeclaratorSyntax in variableList)
+                {
+                    boundDeclarations[i++] = BindVariableDeclaration(kind, isVar, variableDeclaratorSyntax, typeSyntax, declType, alias, diagnostics);
```

Comment:
>BindVariableDeclaration [](start = 45, length = 23)

We're calling `BindVariableDeclaration` twice on each `using` declaration - once here and again in `if` statement below. Is the second bind necessary?

Also, we're dropping `disposeMethod` and `iDisposableConversion` in the second bind when there are multiple variables. Is that intended?

Reply:
This PR is only supposed to take care of the single variable case, where you can see that it is passed on. Julian preferred I do the multiple variable in a different PR. Will fix double-call now.